### PR TITLE
Enemy ai chasing state

### PR DESCRIPTION
**Pull request by MysticAmazon**
everything was cleaned up , i did update from main before this and had some conflicts so i used the updated versions from main for i believe it was 2 scripts, logic works though

### CODE REVIEW

File: Source/SteveSloth/SlingshotAmmoBaseClass.h
Comment (Saretzky) [inline Source/SteveSloth/SlingshotAmmoBaseClass.h:1]: uncomment this out

File: Source/SteveSloth/Private/MyEnemyAIChasingState.cpp
Comment (Saretzky) [inline Source/SteveSloth/Private/MyEnemyAIChasingState.cpp:1]: delete this and the .cpp

File: Content/Blueprints/TestEnemy.uasset
Comment (Saretzky) [inline Content/Blueprints/TestEnemy.uasset:1]: delete this

File: Content/Levels/LevelOne.umap
Comment (Saretzky) [inline Content/Levels/LevelOne.umap:1]: dont need to include this

**Review by Saretzky — CHANGES_REQUESTED** [review]

File: Content/Blueprints/TestEnemy.uasset
Comment (MysticAmazon) [reply Content/Blueprints/TestEnemy.uasset:1]: i have this only because the original one is not working i had to create a new one do i still delete it?

Comment (Saretzky) [reply Content/Blueprints/TestEnemy.uasset:1]: yeah, delete test enemy

File: Content/Levels/LevelOne.umap
Comment (MysticAmazon) [reply Content/Levels/LevelOne.umap:1]: i dont see this in my commit

Comment (MysticAmazon) [reply Content/Levels/LevelOne.umap:1]: made all other changes though

**Review by Jeff-Moreau — CHANGES_REQUESTED** [review]
gonna have to find a way to get rid of TestEnemy and LevelOne from the commit

**Review by Jeff-Moreau — APPROVED** [review]
Looks good

**Review by Saretzky — APPROVED** [review]
looks good

**Pull request merged**